examples/oft-aptos/aptos.layerzero.config.ts

```diff
@@ -0,0 +1,161 @@
+import { EndpointId } from '@layerzerolabs/lz-definitions-v3'
+import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities-v3'
+import { bscConfig } from './evmOAppConfig'
+import { ethConfig } from './evmOAppConfig'
+
+import type { OAppOmniGraphHardhat, OmniPointHardhat } from '@layerzerolabs/toolbox-hardhat'
+
+const ethereumContract: OmniPointHardhat = {
+    eid: EndpointId.ETHEREUM_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const bscContract: OmniPointHardhat = {
+    eid: EndpointId.BSC_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const aptosContract: OmniPointHardhat = {
+    eid: EndpointId.APTOS_V2_TESTNET,
+    contractName: 'oft',
+}
+
+const ethereumSandboxContract: OmniPointHardhat = {
+    eid: EndpointId.ETHEREUM_V2_SANDBOX,
+    contractName: 'MyOFT',
+}
+
+const bscSandboxContract: OmniPointHardhat = {
+    eid: EndpointId.BSC_V2_SANDBOX,
+    contractName: 'MyOFT',
+}
+
+const aptosSandboxContract: OmniPointHardhat = {
+    eid: EndpointId.APTOS_V2_SANDBOX,
+    contractName: 'oft',
+}
+
+const config: OAppOmniGraphHardhat = {
+    contracts: [
+        {
+            contract: ethereumContract,
+            config: ethConfig.accountConfig,
+        },
+        {
+            contract: bscContract,
+            config: bscConfig.accountConfig,
+        },
+        {
+            contract: ethereumSandboxContract,
+            config: ethConfig.accountConfig,
+        },
+        {
+            contract: bscSandboxContract,
+            config: bscConfig.accountConfig,
+        },
+        {
+            contract: aptosContract,
+            config: {
+                delegate: '58b730d07e98a22f2b357bee721115c986e4dc873c1884763708ee3d4006f74e',
+                owner: '58b730d07e98a22f2b357bee721115c986e4dc873c1884763708ee3d4006f74e',
+            },
+        },
+    ],
+    connections: [
+        {
+            from: aptosContract,
+            to: bscContract,
+            config: {
+                enforcedOptions: [
+                    {
+                        msgType: 1,
+                        optionType: ExecutorOptionType.LZ_RECEIVE,
+                        gas: 1000, // gas limit in wei for EndpointV2.lzReceive
```

Comment:
1000 gas when sending to BSC? It seems wrong. Seems like we should use a value here like 80k or something.

Reply:
yeah that was just a placeholder